custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html

```diff
@@ -1,6 +1,9 @@
 {% extends "pact/patient/pactpatient_base.html" %}
 {% load case_tags %}
+{% load hq_shared_tags %}
+
 {% block patient-tab-container %}
+<script src="{% new_static 'jquery-timeago/jquery.timeago.js' %}"></script>
```

Comment:
Given that this library isn't used often, putting it in the reports themselves sounds preferable to me.

Probably better to do it now (remove it from B2 base and add it to the reports that need it) than to remember as each report is upgraded.